### PR TITLE
Add toStack to Ordered primitive Iterable

### PR DESCRIPTION
**Pull request by DineshPurushothamacharya**
Signed-off-by: Dinesh Purushothamacharya <Dinesh.Purushothamacharya@cerner.com>

### CODE REVIEW

**Comment by DineshPurushothamacharya** [comment]
Hello @donraab could you please review and merge this pull request.

**Comment by donraab** [comment]
> Hello @donraab could you please review and merge this pull request.

Hi @DineshPurushothamacharya, yes, before I give a quick review again, can you update the commit message and PR message... they both have a log of a lot of activity and other unimportant stuff.  Thanks!

**Review by donraab — CHANGES_REQUESTED** [review]
Hi @DineshPurushothamacharya, I have one minor change requested inline. Also, if you can clean up the commit and PR comments as they have a log of all the changes with extra unnecessary info. Thanks for being so patient!

**Comment by donraab** [comment]
HI @DineshPurushothamacharya, I am ready to merge once you make these two small changes. I am not going to merge anything else until this PR is merged. Thanks!

File: eclipse-collections-code-generator/src/main/resources/test/stack/immutable/immutablePrimitiveArrayStackTest.stg
Comment (DineshPurushothamacharya) [reply eclipse-collections-code-generator/src/main/resources/test/stack/immutable/immutablePrimitiveArrayStackTest.stg]: Done.

**Comment by DineshPurushothamacharya** [comment]
> HI @DineshPurushothamacharya, I am ready to merge once you make these two small changes. I am not going to merge anything else until this PR is merged, because you've been patient and waited long enough. Thanks!

Hi @donraab I have made the changes as you have requested. Please check.

**Review by donraab — APPROVED** [review]
LGTM! Thank you for the contribution and your patience @DineshPurushothamacharya! I will merge once the builds pass.

**Pull request merged**
